tools/StaticAnalysis/Exceptions/BreakingChangeIssues.csv

```diff
@@ -787,4 +787,45 @@
+"D:\azpwrEHDR\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.EventHub\Microsoft.Azure.Commands.EventHub.dll","Microsoft.Azure.Commands.EventHub.Commands.GetAzureEventHubAuthorizationRule","Get-AzureRmEventHubAuthorizationRule","0","2100","The parameter 'ResourceGroupName' in cmdlet 'Get-AzureRmEventHubAuthorizationRule' is no longer in the parameter set 'NamespaceAuthorizationRuleSet'.","Add parameter 'ResourceGroupName' back to the parameter set 'NamespaceAuthorizationRuleSet'."
+"D:\azpwrEHDR\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.EventHub\Microsoft.Azure.Commands.EventHub.dll","Microsoft.Azure.Commands.EventHub.Commands.GetAzureEventHubAuthorizationRule","Get-AzureRmEventHubAuthorizationRule","0","2100","The parameter 'Name' in cmdlet 'Get-AzureRmEventHubAuthorizationRule' is no longer in the parameter set 'NamespaceAuthorizationRuleSet'.","Add parameter 'Name' back to the parameter set 'NamespaceAuthorizationRuleSet'."
+"D:\azpwrEHDR\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.EventHub\Microsoft.Azure.Commands.EventHub.dll","Microsoft.Azure.Commands.EventHub.Commands.GetAzureEventHubAuthorizationRule","Get-AzureRmEventHubAuthorizationRule","0","2100","The parameter 'DefaultProfile' in cmdlet 'Get-AzureRmEventHubAuthorizationRule' is no longer in the parameter set 'NamespaceAuthorizationRuleSet'.","Add parameter 'DefaultProfile' back to the parameter set 'NamespaceAuthorizationRuleSet'."
+"D:\azpwrEHDR\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.EventHub\Microsoft.Azure.Commands.EventHub.dll","Microsoft.Azure.Commands.EventHub.Commands.GetAzureEventhubKey","Get-AzureRmEventHubKey","0","1020","The cmdlet 'Get-AzureRmEventHubKey' no longer has output type 'Microsoft.Azure.Commands.EventHub.Models.ListKeysAttributes'.","Make cmdlet 'Get-AzureRmEventHubKey' return type 'Microsoft.Azure.Commands.EventHub.Models.ListKeysAttributes'."
+"D:\azpwrEHDR\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.EventHub\Microsoft.Azure.Commands.EventHub.dll","Microsoft.Azure.Commands.EventHub.Commands.GetAzureEventhubKey","Get-AzureRmEventHubKey","0","2100","The parameter 'Namespace' in cmdlet 'Get-AzureRmEventHubKey' is no longer in the parameter set 'NamespaceAuthorizationRuleSet'.","Add parameter 'Namespace' back to the parameter set 'NamespaceAuthorizationRuleSet'."
+"D:\azpwrEHDR\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.EventHub\Microsoft.Azure.Commands.EventHub.dll","Microsoft.Azure.Commands.EventHub.Commands.GetAzureEventhubKey","Get-AzureRmEventHubKey","0","2100","The parameter 'ResourceGroupName' in cmdlet 'Get-AzureRmEventHubKey' is no longer in the parameter set 'NamespaceAuthorizationRuleSet'.","Add parameter 'ResourceGroupName' back to the parameter set 'NamespaceAuthorizationRuleSet'."
```

Comment:
You didn't change ResourceGroupName attribute, so this shouldn't cause a failure.

Reply:
reverted the changes and only exceptions caused due to PS prefix to Models are added